src/allmydata/client.py

```diff
@@ -533,35 +530,35 @@ def init_web(self, webport):
         self.log("init_web(webport=%s)", args=(webport,))
 
         from allmydata.webish import WebishServer
-        nodeurl_path = os.path.join(self.basedir, "node.url")
-        staticdir_config = self.get_config("node", "web.static", "public_html").decode("utf-8")
-        staticdir = abspath_expanduser_unicode(staticdir_config, base=self.basedir)
+        nodeurl_path = self.config.get_config_path("node.url")
+        staticdir_config = self.config.get_config("node", "web.static", "public_html").decode("utf-8")
+        staticdir = self.config.get_config_path(staticdir_config)
         ws = WebishServer(self, webport, nodeurl_path, staticdir)
         self.add_service(ws)
 
     def init_ftp_server(self):
-        if self.get_config("ftpd", "enabled", False, boolean=True):
+        if self.config.get_config("ftpd", "enabled", False, boolean=True):
             accountfile = from_utf8_or_none(
-                self.get_config("ftpd", "accounts.file", None))
+                self.config.get_config("ftpd", "accounts.file", None))
             if accountfile:
-                accountfile = abspath_expanduser_unicode(accountfile, base=self.basedir)
-            accounturl = self.get_config("ftpd", "accounts.url", None)
-            ftp_portstr = self.get_config("ftpd", "port", "8021")
+                accountfile = self.config.get_config_path(accountfile)
```

Comment:
Another `abspath_expanduser_unicode` vs `get_config_path` case...

Reply:
get_config_path is doing the expand now